R/data-importer-configuration.R

```diff
@@ -8,9 +8,17 @@ DataImporterConfiguration <- R6::R6Class(
   inherit = DotNetWrapper,
   cloneable = TRUE,
   active = list(
+    #' @field ref Underlying .NET reference
```

Comment:
This is an example where we have to change the `ref` of the object, so we overload the method.

Reply:
I don't see how this is a good idea!

WHY ?

Reply:
Otherwise we must change the constructor. Currently it gets a path to a configuration file, but not a `ref`. So either we change it to receiving a `ref` - but in this case creating from a file must be implemented through a function - or we allow to set ref from ouside.

OR the constructor can receive a `ref` or a path to the file - which I do not like at all.

Reply:
i need to see how this is being used. I don't understand. I'll check it out. Please do not merge

Reply:
ok so there is definitely no need for this This is used once, where the DataImportConfiguraiton object is instantiated. 
Like we do for many other objects, pass it in the constructor so that no one will ever create this object by hand but always use the service

Reply:
@msevestre OK then I will change the constructor to get a `ref` as argument instead of file path to load from, and create a function `loadDataImporterConfiguration`, agreed? This will be a breaking change but probably it was never used before anyways.

Reply:
yes agreed

Reply:
@msevestre Implemented.